doc/flatpak-override.xml

```diff
@@ -51,7 +51,7 @@
             <command>flatpak override</command>.
         </para>
         <para>
-            The application overrides are saved in text files residing in $XDG_DATA_HOME/flatpak/overrides in user mode. 
+            The application overrides are saved in text files residing in $XDG_DATA_HOME/flatpak/overrides in user mode.
```

Comment:
Please don't edit unrelated lines in a commit that should be doing one thing. If you want to normalize the man pages to have no trailing whitespace, that could be a a separate PR.